subsys/spm/secure_services.c

```diff
@@ -47,6 +50,34 @@ int spm_secure_services_init(void)
 }
 
 
+#ifdef CONFIG_SPM_SERVICE_READ
+__TZ_NONSECURE_ENTRY_FUNC
+int spm_request_read(void *destination, const void *source, size_t num)
+{
+	static const u32_t read_ranges[] = {
```

Comment:
why not array of struct with begin and end addresses?

Reply:
See proposal

Reply:
goog